.circleci/config.yml

```diff
@@ -800,101 +488,13 @@ workflows:
             tags:
               only: /.*/
 
-      - download_drbuddi_rpe_series:
-          requires:
-            - build
-          filters:
-            tags:
-              only: /.*/
-
-      - download_drbuddi_epi:
-          requires:
-            - build
-          filters:
-            tags:
-              only: /.*/
-
       - download_DSDTI:
           requires:
             - build
           filters:
             tags:
               only: /.*/
 
-      - download_DSCSDSI:
-          requires:
-            - build
-          filters:
-            tags:
-              only: /.*/
-
-      - download_twoses:
-          requires:
-            - build
-          filters:
-            tags:
-              only: /.*/
-
-      - DSCSDSI:
-          requires:
-            - download_DSCSDSI
-          filters:
-            branches:
-              ignore:
-                - /recon\/.*/
-            tags:
-              only: /.*/
-
-      - DRBUDDI_SHORELine_EPI:
-          requires:
-            - download_drbuddi_epi
-          filters:
-            branches:
-              ignore:
-                - /recon\/.*/
-            tags:
-              only: /.*/
-
-      - DRBUDDI_eddy_rpe_series:
-          requires:
-            - download_drbuddi_rpe_series
-          filters:
-            branches:
-              ignore:
-                - /recon\/.*/
-            tags:
-              only: /.*/
-
-      - DRBUDDI_TENSORLine_EPI:
-          requires:
-            - download_DSDTI
-          filters:
-            branches:
-              ignore:
-                - /recon\/.*/
-            tags:
-              only: /.*/
-
-      - DSDTI_nofmap:
-          requires:
-            - download_DSDTI
-          filters:
-            branches:
-              ignore:
-                - /recon\/.*/
-            tags:
-              only: /.*/
-
-      - DSDTI_synfmap:
-          requires:
-            - download_DSDTI
-          filters:
-            branches:
-              ignore:
-                - /recon\/.*/
-            tags:
-              only: /.*/
-
       - DSDTI_TOPUP:
```

Comment:
DSDTI_TOPUP was a test for the prep-then-recon workflow. It isn't possible anymore so it should go